app/services/idv/gpo_mail.rb

```diff
@@ -41,39 +43,26 @@ def last_not_too_recent_enabled?
       IdentityConfig.store.minimum_wait_before_another_usps_letter_in_hours != 0
     end
 
-    attr_reader :current_user
-
     def too_many_letter_requests_within_window?
       return false unless window_limit_enabled?
-
-      number_of_letter_requests_within(
-        IdentityConfig.store.max_mail_events_window_in_days.days,
-        maximum: IdentityConfig.store.max_mail_events,
-      ) >= IdentityConfig.store.max_mail_events
+      current_user.gpo_confirmation_codes.where(
+        updated_at: Range.new(
+          IdentityConfig.store.max_mail_events_window_in_days.days.ago,
+          Time.zone.now,
+        ),
+      ).count > IdentityConfig.store.max_mail_events
```

Comment:
this was `>=` before, was the change intentional?

Reply:
It was not. Nice catch!

Reply:
Fun fact: This one-character mistake caused 3 separate test files to fail. Fix incoming.